ci/scripts/e2e-source-test.sh

```diff
@@ -44,11 +47,30 @@ echo "--- Prepare RiseDev dev cluster"
 cargo make pre-start-dev
 cargo make link-all-in-one-binaries
 
+echo "--- e2e, ci-1cn-1fe, cdc source"
+# install mysql client
+apt-get -y install mysql-client
+# import data to mysql
+mysql --host=mysql --port=3306 -u root -p123456 < ./e2e_test/source/cdc/mysql_cdc.sql
+# start risingwave cluster
+cargo make ci-start ci-1cn-1fe
+# start cdc connector node
+nohup java -cp ./connector-service.jar com.risingwave.sourcenode.service.SourceServiceMain > .risingwave/log/connector-source.log 2>&1 &
```

Comment:
In this future when we combine the sink and source service into a single service, we only need `java -jar ./connector-service.jar`.